### PR TITLE
Updated twitter logo to it's latest logo (X)

### LINKED ISSUE #5149: Update Twitter logo to its latest logo (X)
Description-
We should update the logo of Twitter to its latest one. As it is good to stay updated with the latest changes in the tech world.

image

Here, we can see the logo is still the old one. 
On the page - https://rxdb.info/

### PR DESCRIPTION
**Pull request by niranjan-kurhade**
## Describe the problem you have without this PR
This PR fixes #5149 

In this PR, I have updated the twitter logo to its latest logo.

Screenshots-
image

image

### CODE REVIEW

**Comment by niranjan-kurhade** [comment]
Hey @pubkey, In this PR I have made changes in the `docs-src` folder and not in the `docs` folder. Please review.

File: docs-src/files/icons/twitter-blue.svg
Comment (pubkey) [inline docs-src/files/icons/twitter-blue.svg:1]: Hi. Shouldnt this be blue instead of white? Or maybe does github render the svg wrongly?

Comment (pubkey) [reply docs-src/files/icons/twitter-blue.svg:1]: Aah. I found out why. Twitter is no longer "blue". It is now black and white :)

**Comment by pubkey** [comment]
Thank you very much. Merged.

**Pull request merged**